content/post/articles/Music-and-silence.md

```diff
@@ -0,0 +1,57 @@
+The benefits of music on our brain are numerous we can mention:
+###### Boost motivation
+
+I'm sure you know this good old mister "bug" that everyone wants to embrace he only brings us happiness when coding 🐛, more seriously as a developer, we often if not always encounter bugs in our code which can sometimes push us to abandon our code, because we cannot find the error, or because we consider that we have sought too much for the solution, in short it demotivates us but thanks to the studies cited above we can say that to listen a music that we like can help to take a step back, to reconsider the problem from another angle because the music will make us happy, more relaxed and therefore open to new possibilities to kill or permanently solve this dear mister bug 😊.
+
+###### Boost creativity
+
+Creativity can be boosted by music, according to a [study](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0182210) by researchers at Radboud University (Netherlands) and Sydney University of Technology (Australia) the researchers found that listening to happy music, which they define as music with positive, awakening values (a relative positivity because coders who love devil metal are also happy) facilitates more inspiring creative thinking than silence.
+
+I'm a supporter of this opinion, so here are some nuggets(without wanting to advertise) I use when I meet mister bug:
+- [RISE (ft. The Glitch Mob, Mako, and The Word Alive)](https://www.youtube.com/watch?v=fB8TyLTD7EE)
+- [Phoenix (avec Cailin Russo et Chrissy Costanza)](https://www.youtube.com/watch?v=i1IKnWDecwA)
+- [Take Over (avec Jeremy McKinnon (A Day To Remember), MAX, Henry)](https://www.youtube.com/watch?v=KbNL9ZyB49c)
+
+# Code with silence
+
+Lao Tzu said <<*Silence is a source of Great Strength*>>
+
+Not everyone uses music to cope with the different difficulties that one can encounter while coding so there are those who are for silence (the zone), a 2013 [study](https://www.researchgate.net/publication/259110014_Is_silence_golden_Effects_of_auditory_stimuli_and_their_absence_on_adult_hippocampal_neurogenesis) found that two hours of silence could create new cells in the hippocampus region, a brain area linked to learning, remembering, and emotions.According to a 2006 [study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1860846/), two minutes of silence relieves tension in the body and brain and is more relaxing than listening to music.  
+Silence also has its advantages:
+
+###### Boost creativity
+When you allow thoughts to go where they want, inspiration can flow. Solutions to current or long standing problems may suddenly appear to you, or a workaround or innovative approach may seem more achievable. Ideas for going in a different direction might merge together, help build momentum and enthusiasm to turn them into other potential avenues to pursue.Follow this [link](https://psychcentral.com/blog/the-hidden-benefits-of-silence#1) to read the article.  
```

Comment:
A so-called scientific website with ads all over the place makes me question his legitimacy, moreover it starts with the quote:

> “Silence is a source of great strength.” – Lao Tzu

That we already established is not from Lao Tzu. It makes me question how deep they went in their research. Nonetheless, to their credit, they say:

> Silence may help in the following areas

"may" is a keyword here, that means it's possible, not for sure, but possible. Moreover, nowhere in the article they talk about silence during coding. There, silence is more associated to a break, which is another subject by its own.